File : ./logs/fun6.log is in split mode.
Please run at project root: "python3 fileSplitMerge.py --merge" 